File: app/src/main/java/com/example/culturaverde/LoginMainFragment.java
package com.example.culturaverde;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;


import androidx.fragment.app.Fragment;
import androidx.navigation.Navigation;

public class LoginMainFragment extends Fragment{

    @Override
    public View  onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState){

        return inflater.inflate(R.layout.fragment_login_main, container, false);
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        Button botonplayconsumidores = view.findViewById(R.id.botonPlayConsumidores);

        botonplayconsumidores.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Navigation.findNavController(view).navigate(R.id.action_loginmainfragment_to_principalconsumidoresFragment);
            }
        });
    }
}

File: app/src/main/java/com/example/culturaverde/loginmainfragment.kt
package com.example.culturaverde

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import kotlinx.android.synthetic.main.fragment_loginmain.*

class loginmainfragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_loginmain, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        botonPlayConsumidores.setOnClickListener{

            val action = loginmainfragmentDirections.actionLoginmainfragmentToMenudesplegableConsumidores()
            findNavController().navigate(action)

        }

        botonPlayProductores.setOnClickListener{

            val action = loginmainfragmentDirections.actionLoginmainfragmentToMenudesplegableProductores()
            findNavController().navigate(action)
        }

    }

}
